pkg/compact/compact.go

```diff
@@ -394,58 +394,11 @@ func (cg *Group) Add(meta *metadata.Meta) error {
 	return nil
 }
 
-// IDs returns all sorted IDs of blocks in the group.
-func (cg *Group) IDs() (ids []ulid.ULID) {
-	cg.mtx.Lock()
-	defer cg.mtx.Unlock()
-
-	for id := range cg.blocks {
-		ids = append(ids, id)
-	}
-	sort.Slice(ids, func(i, j int) bool {
-		return ids[i].Compare(ids[j]) < 0
-	})
-	return ids
-}
-
-// MinTime returns the min time across all group's blocks.
-func (cg *Group) MinTime() int64 {
-	cg.mtx.Lock()
-	defer cg.mtx.Unlock()
-
-	min := int64(0)
-	for _, b := range cg.blocks {
-		if b.MinTime < min {
-			min = b.MinTime
-		}
-	}
-	return min
-}
-
-// MaxTime returns the max time across all group's blocks.
-func (cg *Group) MaxTime() int64 {
-	cg.mtx.Lock()
-	defer cg.mtx.Unlock()
-
-	max := int64(0)
-	for _, b := range cg.blocks {
-		if b.MaxTime < max {
-			max = b.MaxTime
-		}
-	}
-	return max
```

Comment:
This always returns 0.

Reply:
You are right - Let's fix it. max := int64(0) should be max := math.MinInt64 Also it should be ` b.MaxTime > max`

I don't think it was well implemented so wonder how those were used....  cc @pracucci 

Let's add tests as well? (:

Reply:
They're definitely wrong 🤔  I will double check it where we use them. Thanks!